src/Task.cc

```diff
@@ -721,6 +720,9 @@ void Task::on_syscall_exit_arch(int syscallno, const Registers& regs) {
           break;
         }
         case PTRACE_ARCH_PRCTL: {
```

Comment:
I think you mean Arch::PTRACE_ARCH_PRCTL here.

Reply:
thanks